Kinvey/Kinvey/Client.swift

```diff
@@ -267,6 +273,7 @@ open class Client: Credential {
         if let user = keychain.user {
             user.client = self
             activeUser = user
+            clientId = keychain.clientId
             let customUser = user as! U
             completionHandler(.success(customUser))
         } else if let kinveyAuth = sharedKeychain?.kinveyAuth {
```

Comment:
right, we are storing the client_id in the keychain, but if the user is using SSO it would be a shared keychain, so my question is if we should allow the user to specify a `client_id` in this case.